lib/fetchBlob/azure/index.native.js

```diff
@@ -4,11 +4,15 @@ import config from '../../config';
 import ReverMediaError from '../../ReverMediaError';
 
 export default async function fetchFromAzure(args = {}) {
-  const { url } = args;
+  const { url, fileExtension } = args;
   const headers = buildHeaders(args);
 
   try {
-    const response = await RNFetchBlob.fetch('GET', url, headers);
+    const response = await RNFetchBlob.config({ fileCache: true, appendExt: fileExtension }).fetch(
```

Comment:
This `config` method receives the parameters to save files in cache. Please note that `fileExtension` is required to properly generate the file path.
Here's the related API documentation https://github.com/joltup/rn-fetch-blob/wiki/Classes#user-content-rnfetchblobconfig